rpc/json.go

```diff
@@ -18,71 +18,111 @@ package rpc
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
+	"net"
 	"reflect"
-	"strconv"
 	"strings"
 	"sync"
-
-	"github.com/ethereum/go-ethereum/log"
+	"time"
 )
 
 const (
-	jsonrpcVersion           = "2.0"
+	vsn                      = "2.0"
```

Comment:
Three letter abbreviation of `version`, really? Please use `version` or leave it as it was previously

Reply:
IMHO the constant name has to be shorter than `"2.0"`, otherwise it's not worth defining it.

Reply:
I will change it to `version`.